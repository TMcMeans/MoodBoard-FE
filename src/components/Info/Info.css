/*COMPONENT STYLES*/
.Info {
  background-color: #2A2A2A;
  font-family: 'Raleway', sans-serif;
  height: 100vh;
  margin: 0;
  color: red;
  text-align: center;
}
.info-heading{
  color: white;
}

.info-paragraph{
  color: white;
  margin-left: 50px;
  margin-right: 50px;

}
