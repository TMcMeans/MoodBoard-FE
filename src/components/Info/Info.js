import React, { Component } from 'react';
import { Switch, Route, withRouter } from 'react-router-dom';

import './Info.css';

class Info extends Component {
  render(){
    return(
      <div>
        <h1 className ="info-heading"> Welcome to MoodBoard. </h1>
        <p className = "info-paragraph"> MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a userâ€™s projected mood on any given day. </p>
      </div>

    );
  }
}




export default withRouter(Info);
