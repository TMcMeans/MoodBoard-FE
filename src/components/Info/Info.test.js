import React from 'react';
import { shallow } from 'enzyme';
import { Info } from './Info';

describe('Info', () => {
  it('should render a snapshot with correct data passed in', () => {
    const wrapper = shallow(<Info />);
    expect(wrapper).toMatchSnapshot();
  });
});
