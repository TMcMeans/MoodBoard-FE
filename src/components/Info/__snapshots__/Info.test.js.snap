// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info should render a snapshot with correct data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Info />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Logo />,
        <h1
          className="info-heading"
        >
           About 
        </h1>,
        <p
          className="info-paragraph"
        >
           
          MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.
           
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " About ",
          "className": "info-heading",
        },
        "ref": null,
        "rendered": " About ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            "MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.",
            " ",
          ],
          "className": "info-paragraph",
        },
        "ref": null,
        "rendered": Array [
          " ",
          "MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.",
          " ",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Logo />,
          <h1
            className="info-heading"
          >
             About 
          </h1>,
          <p
            className="info-paragraph"
          >
             
            MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.
             
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " About ",
            "className": "info-heading",
          },
          "ref": null,
          "rendered": " About ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              "MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.",
              " ",
            ],
            "className": "info-paragraph",
          },
          "ref": null,
          "rendered": Array [
            " ",
            "MoodBoard is a daily journaling tool, record of personal reflections, and long-term reference for a user’s projected mood on any given day.",
            " ",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
